CMakeLists.txt

```diff
@@ -247,7 +254,11 @@ else()
                              ${ArrayFire_SOURCE_DIR}/include
                              ${ArrayFire_BINARY_DIR}/include
                              ${ArrayFire_SOURCE_DIR}/src/backend)
-  target_link_libraries(bin2cpp PRIVATE spdlog)
+  if(TARGET spdlog::spdlog_header_only)
```

Comment:
af_spdlog?

Also why is bin2cpp using spdlog anyway. It doesn't look like it uses it.

Reply:
bin2cpp uses `common/util.hpp` and that indirectly causes this I think.